packages/field-base/src/input-mixin.js

```diff
@@ -267,19 +267,5 @@ export const InputMixin = dedupingMixin(
         // Setting a value programmatically, sync it to input element.
         this._forwardInputValue(newVal);
       }
-
-      /**
-       * Sets the `_hasInputValue` property based on the `input` event.
-       *
-       * @param {InputEvent} event
-       * @protected
-       */
-      _setHasInputValue(event) {
```

Comment:
Let's also remove this method from `input-mixin.d.ts`.

Reply:
Done.